crates/sui-config/src/node.rs

```diff
@@ -233,11 +233,17 @@ pub struct ExecutionTimeObserverConfig {
     /// If unspecified, this will default to `50_000`.
     pub object_utilization_cache_size: Option<NonZeroUsize>,
 
-    /// If true, the execution time observer will report per-object utilization metrics.
-    /// Warning: This metric may have very large cardinality.
+    /// If true, the execution time observer will report per-object utilization metrics
+    /// with full object IDs. When set, the metric can have a high cardinality, so this
+    /// should not be used except in controlled tests where there are a small number of
+    /// objects.
+    ///
+    /// If false, object utilization is reported using hash(object_id) % 16 as the key,
+    /// which still allows observation of utilization when there are small numbers of
+    /// over-utilized objects.
     ///
     /// If unspecified, this will default to `false`.
-    pub report_object_utilization_metric: Option<bool>,
+    pub report_object_utilization_metric_with_full_id: Option<bool>,
```

Comment:
I don't think this change is safe for existing configs? maybe we don't care though?

Reply:
Yeah I assume that no one is using this outside of your sui-ops branch?